testsuite/bridged_interactor_noflush/init.yml

```diff
@@ -0,0 +1,9 @@
+unbuffered: True
```

Comment:
BRO

Reply:
if it make you feel worse, there's one that says `unbuffered: yes`

Reply:
It doesn't, because `yes` is actually part of the YAML spec.

Reply:
big TIL moment